### DISCUSSION TITLE
nuxt 3.2.3 and 3.2.2 has crazy slow cold start times

### DISCUSSION THREAD

**Discussion by jpengelbrecht** [nuxt/nuxt#19358] [Questions]
We recently upgraded from 3.2.1 to 3.2.2 and then to 3.2.3 however we noticed that since the version upgrade npm install and starting up the dev server takes immensely long.   i am talking about a solid 10 minutes of waiting.   has anyone else had this problem with the upgrade? 
we have noticed the m1 chip macs load up quite fast but the i9 ones takes alot longer.   in the below screenshot you can see a` npm run dev
` command takes 5 mins before the command even starts.  on previous versions it was instantaneous 
![image](https://user-images.githubusercontent.com/93583336/222106804-7a42eeb9-2e65-4d3d-849c-dfff50167367.png)

**Comment by AloisSeckar** [comment]
I just update my project to `3.2.3` and experience no performance issues. Dev server starts as usual and also CI/CD pipeline took same duration as before. So it doesn't look like a problem per se.

Might be together with some other dependencies or due to some specific setup of your project. Do you mind posting your `package.json` and `nuxt.config.ts` so Nuxt team members can have a look?

**Reply by jpengelbrecht** [reply]
so for privacy reasons i cant add those files.   I mainly just wanted to add this discussion to see if anyone add simliar problems

**Comment by dsvgl** [comment]
Have you tried to delete `node_modules` and also recreate your lock file?

**Reply by jpengelbrecht** [reply]
jip no affect

**Comment by jpengelbrecht** [comment]
so we fixed the issue we were importing 
```
import {
  ApolloClient,
  InMemoryCache,
  HttpLink,
  from,
} from "@apollo/client/core";
```
in our nuxt config.   

it seems that this grinds the build to a halt on some computers for some reason,.
